devApps/XFormsApp/SecondPage.cs

```diff
@@ -27,12 +27,14 @@
 
 using Microsoft.IdentityModel.Clients.ActiveDirectory;
 using System;
+using System.Diagnostics.CodeAnalysis;
 using System.Text;
 using Xamarin.Forms;
 using Xamarin.Forms.PlatformConfiguration;
 
 namespace XFormsApp
 {
+    [SuppressMessage("AsyncUsage.CSharp.Reliability", "AvoidAsyncVoid", Justification = "Reviewed.")]
```

Comment:
>[SuppressMessage("AsyncUsage.CSharp.Reliability", "AvoidAsyncVoid", Justification = "Reviewed.")] [](start = 4, length = 97)

see similar comment...

Reply:
globalsuppressions and also relevant justification

---
In reply to: [168047125](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/992#discussion_r168047125) [](ancestors = 168047125)